app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+import os
+from models import Actor
+
+
+ABSOLUTE_PATH = os.path.abspath("../cinema.sqlite")
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect(ABSOLUTE_PATH)
```

Comment:
Consider handling exceptions that might occur during the database connection process. This can help in debugging and ensure that the application fails gracefully if the database is not accessible.